tests/unit/CFTypeDetectorTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * LICENSE
+ *
+ * This file is part of CFPropertyList.
+ *
+ * The PHP implementation of Apple's PropertyList can handle XML PropertyLists
+ * as well as binary PropertyLists. It offers functionality to easily convert
+ * data between worlds, e.g. recalculating timestamps from unix epoch to apple
+ * epoch and vice versa. A feature to automagically create (guess) the plist
+ * structure from a normal PHP data structure will help you dump your data to
+ * plist in no time.
+ *
+ * Copyright (c) 2018 Teclib'
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * ------------------------------------------------------------------------------
+ * @author    Christian Kruse <cjk@wwwtech.de>
+ * @copyright Copyright © 2018 Teclib
+ * @package   plist
```

Comment:
The new name is `CFPropertyList`

Reply:
This is not the new name.. this is the name it was when we took the lib. 

but +1, I'll create a PR for that